aip/general/0149.md

```diff
@@ -50,6 +50,15 @@ field behavior as defined in [AIP-203][]. For example, a field labeled with
 `google.api.field_behavior = REQUIRED` if the field must be set. If you only
 want to document the server perceived behavior of a field, read [AIP-203][].
 
+### Backwards compatibility
+
+Adding or removing the `optional` label to an existing field is considered a
+backwards incompatible change in some languages e.g. Go, due to a change in
```

Comment:
You active voice and shorter sentences whenever possible.

> It is a backwards incompatible change to add or remove the `optional` qualifier to an existing field. This is because the compiled src API is changed (in some languages). For example, in Golang, adding `optional` changes the field type of primitives to be the pointer variant of their original type, e.g. a field formerly of type `string` becomes `*string`, etc. Accordingly, this change requires that both clients and servers update their usage of the changed field in unison, which is risky and error prone. Additional information is [documented by Protobuf][proto docs].

Reply:
Thanks! I've updated with your version.